src/views/Register/components/Rules.js

```diff
@@ -0,0 +1,51 @@
+export const email = [
+    {
+        required: true,
+        message: "Email is required",
+        trigger: "blur",
+      },
+      {
+        min: 2,
+        message: "Email length should be at least 3 characters",
+        trigger: "blur",
```

Comment:
this one is just for information.
In this case, it will be always same -> min 2 lenght. Generally speaking, when programming anything, it pays to consider whether you can do something universal that can be used in several places. In this case, it would be possible to separate the rule from the length of the characters into a separate variable with a parameter that would be used everywhere. 

In this case, let it stay as it is, but bear in mind that it could be slightly modified for code improvement